Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -255,6 +266,13 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         tabManager.removeTab(tab)
         tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: UITableViewRowAnimation.Automatic)
     }
+    
+    func closeCurrTab(sender: UIButton) {
+        let indexPath:NSIndexPath = NSIndexPath(forRow: sender.tag, inSection: 0)
+        let tab = tabManager.getTab(indexPath.item)
+        tabManager.removeTab(tab)
+        tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: UITableViewRowAnimation.Automatic)
```

Comment:
UX may want to add an animation here, but we can probably handle this polish in a follow-up bug :)